api/v1beta2/cryostat_types.go

```diff
@@ -625,6 +629,15 @@ type TemplateConfigMap struct {
 	Filename string `json:"filename"`
 }
 
+// A ConfigMap containing a .json automated rule file.
+type AutomatedRuleConfigMap struct {
+	// Name of config map in the local namespace.
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:io.kubernetes:ConfigMap"}
+	ConfigMapName string `json:"configMapName"`
+	// Filename within config map containing the automated rule file.
+	Filename string `json:"filename"`
```

Comment:
This could use a `// +operator-sdk:csv:customresourcedefinitions:type=spec` comment. Would you be able to add it for the Event Templates config map struct too? It'll need a `make bundle` after.